flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -132,13 +133,22 @@ typedef enum {
 	MSP_MAYBE_UAVTALK2,
 	MSP_MAYBE_UAVTALK3,
 	MSP_MAYBE_UAVTALK4,
-	MSP_MAYBE_UAVTALK_SLOW2,
-	MSP_MAYBE_UAVTALK_SLOW3,
-	MSP_MAYBE_UAVTALK_SLOW4,
-	MSP_MAYBE_UAVTALK_SLOW5,
-	MSP_MAYBE_UAVTALK_SLOW6
 } msp_state;
 
+typedef enum __attribute__ ((__packed__)) {
+	PROTOCOL_SIMONK = 0,
+	PROTOCOL_BLHELI = 1,
+	PROTOCOL_KISS = 2,
+	PROTOCOL_KISSALL = 3,
+	PROTOCOL_CASTLE = 4,
+	PROTOCOL_4WAY = 0xff,
+} msp_esc_protocol;
+
+struct msp_cmddata_escserial {
+	msp_esc_protocol protocol;
```

Comment:
I don't understand..

Is your objection the typedef and you'd prefer enum msp_esc_protocol here?  Or do you want me to use uint8_t like a lot of our other stuff does  Or is it variable name?

Things are already fairly (descriptively) buried in `m->cmd_data.escserial.protocol` --  that's why I rejected making it `m->cmd_data.cmd_escserial.esc_protocol`